packages/dds/tree/src/core/revertible.ts

```diff
@@ -47,6 +47,8 @@ export interface RevertibleAlpha extends Revertible {
 	/**
 	 * Clones the {@link Revertible} to a target branch.
 	 *
+	 * @remarks To clone `RevertibleAlpha` in batch, use {@link cloneRevertibles}.
```

Comment:
Let's move away from the "batch" terminology for the same reason we're not using it in the name anymore. I think this would be sufficient:

```suggestion
	 * @remarks To clone a group of`RevertibleAlpha`s, use {@link cloneRevertibles}.
```